### ISSUE TITLE
Results about VehicleID Dataset

### ISSUE THREAD

**Issue by SikaStar** [Confusezius/Deep-Metric-Learning-Baselines#1]
Hi,
Have you tested these deep metric learning methods on VehicleID? What about the results?

**Comment by Confusezius** [comment]
Hi there!

Since Vehicle-ID is currently not as widely used and not easily accessible (and quite large), I haven't run replicable fixed seed runs (yet) on this dataset, hence why they haven't been included. 

The training code/dataloader is set up to work directly with this dataset though, and in `sample_training_runs.sh` you have (commented out) sample training commands to achieve good results on this dataset.

As a reference however, for the small and large test set, ResNet with margin loss currently achieves about R@1/R@5 of ~85/~92 and ~80/~89 respectively, with NPair and semihard trailing a bit behind. 

Hope this helps! If I have time in the future, I will include these results as well.
